lib/preflight/preflighttest.ts

```diff
@@ -21,48 +21,42 @@ const DEFAULT_TEST_DURATION = 10 * SECOND;
  * progress values that are sent by {@link PreflightTest#event:progress}
  * @enum {string}
  */
-enum PreflightProgress {
+const PreflightProgress = {
   /**
-   * Preflight test {@link PreflightTest} has successfully acquired media
+   * {@link PreflightTest} has successfully generated synthetic tracks
```

Comment:
The name of the event is 'mediaAcquired', I think it's ok to mention `...has successfully acquired media` right? Mentioning `generated synthetic tracks` might be more confusing to the customer.